runelite-client/src/main/java/net/runelite/client/plugins/bosstimer/Boss.java

```diff
@@ -59,7 +59,8 @@
 	DUSK(NpcID.DUSK_7889, 5, ChronoUnit.MINUTES, ItemID.NOON),
 	ALCHEMICAL_HYDRA(NpcID.ALCHEMICAL_HYDRA_8622, 25200, ChronoUnit.MILLIS, ItemID.IKKLE_HYDRA),
 	SARACHNIS(NpcID.SARACHNIS, 10, ChronoUnit.SECONDS, ItemID.SRARACHA),
-	ZALCANO(NpcID.ZALCANO_9050, 21600, ChronoUnit.MILLIS, ItemID.SMOLCANO);
+	ZALCANO(NpcID.ZALCANO_9050, 21600, ChronoUnit.MILLIS, ItemID.SMOLCANO),
+	MUSPAH(NpcID.PHANTOM_MUSPAH_12080, 30, ChronoUnit.SECONDS, ItemID.MUPHIN);
```

Comment:
Should probably use the full name for the enum, to match how the other bosses are listed
```suggestion
	PHANTOM_MUSPAH(NpcID.PHANTOM_MUSPAH_12080, 30, ChronoUnit.SECONDS, ItemID.MUPHIN);
```